### ISSUE TITLE
[Android] Examples don't run: Error: Activity class {com.example/com.example.MainActivity} does not exist

### ISSUE THREAD

**Issue by darkbasic** [react-native-maps/react-native-maps#3853]
# Bug report

## Summary

The bundled examples fail to run on Android, with the following error:

```
$ npx react-native run-android
info Running jetifier to migrate libraries to AndroidX. You can disable it using "--no-jetifier" flag.
Jetifier found 967 file(s) to forward-jetify. Using 8 workers...
info JS server already running.
info Installing the app...
Downloading https://services.gradle.org/distributions/gradle-6.2-all.zip
.............10%.............20%.............30%..............40%.............50%.............60%.............70%..............80%.............90%.............100%

Welcome to Gradle 6.2!

Here are the highlights of this release:
 - Dependency checksum and signature verification
 - Documentation links in deprecation messages
 - Shareable read-only dependency cache

For more details see https://docs.gradle.org/6.2/release-notes.html

Starting a Gradle Daemon, 1 stopped Daemon could not be reused, use --status for details

> Task :app:installDebug
08:11:08 V/ddms: execute: running am get-config
08:11:08 V/ddms: execute 'am get-config' on 'emulator-5554' : EOF hit. Read: -1
08:11:08 V/ddms: execute: returning
Installing APK 'app-debug.apk' on 'Pixel_3a_API_30_x86(AVD) - 11' for app:debug
08:11:08 D/app-debug.apk: Uploading app-debug.apk onto device 'emulator-5554'
08:11:08 D/Device: Uploading file onto device 'emulator-5554'
08:11:08 D/ddms: Reading file permision of /home/niko/devel/react-native-maps/example/android/app/build/outputs/apk/debug/app-debug.apk as: rw-rw-r--
08:11:08 V/ddms: execute: running pm install -r -t "/data/local/tmp/app-debug.apk"
08:11:08 V/ddms: execute 'pm install -r -t "/data/local/tmp/app-debug.apk"' on 'emulator-5554' : EOF hit. Read: -1
08:11:08 V/ddms: execute: returning
08:11:08 V/ddms: execute: running rm "/data/local/tmp/app-debug.apk"
08:11:08 V/ddms: execute 'rm "/data/local/tmp/app-debug.apk"' on 'emulator-5554' : EOF hit. Read: -1
08:11:08 V/ddms: execute: returning
Installed on 1 device.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.2/userguide/command_line_interface.html#sec:command_line_warnings

BUILD SUCCESSFUL in 1m 2s
46 actionable tasks: 5 executed, 41 up-to-date
info Connecting to the development server...
8081
info Starting the app on "emulator-5554"...
Starting: Intent { cmp=com.example/.MainActivity }
Error type 3
Error: Activity class {com.example/com.example.MainActivity} does not exist.
```

## Environment info

OS is Fedora Linux 34.

I'm using the Android Emulator, which so far worked fine for my react-native projects.

`react-native info` output:

```bash
info Fetching system and libraries information...
System:
    OS: Linux 5.12 Fedora 34 (Workstation Edition) 34 (Workstation Edition)
    CPU: (8) x64 AMD Ryzen 5 2400G with Radeon Vega Graphics
    Memory: 26.77 GB / 62.81 GB
    Shell: 5.1.0 - /bin/bash
  Binaries:
    Node: 14.17.0 - /tmp/yarn--1623867341376-0.3224512875088157/node
    Yarn: 1.22.10 - /tmp/yarn--1623867341376-0.3224512875088157/yarn
    npm: 6.14.13 - ~/.nvm/versions/node/v14.17.0/bin/npm
    Watchman: 0.0.0 - /usr/bin/watchman
  SDKs:
    Android SDK: Not Found
  IDEs:
    Android Studio: Not Found
  Languages:
    Java: Not Found
    Python: 3.9.5 - /usr/bin/python
  npmPackages:
    @react-native-community/cli: Not Found
    react: 16.13.1 => 16.13.1 
    react-native: 0.63.4 => 0.63.4 
  npmGlobalPackages:
    *react-native*: Not Found
Done in 0.86s.
```

## Steps to reproduce

1. git clone this repo
2. cd example
3. npm install
4. npx react-native run-android

Describe what you expected to happen:

1. Starts the app

Describe what you actually happens:

1. `Error: Activity class {com.example/com.example.MainActivity} does not exist`

## Reproducible sample code

The example folder in this repo.

**Comment by darkbasic** [comment]
I've noticed that somehow the app still ends up being loaded in the emulator, but you have to manually open it and at this point seems to work fine. Weird behaviour.

**Comment by alex83130** [comment]
Same issue on my side, I get the same error message

**Comment by ac2-warp** [comment]
This worked for me. https://stackoverflow.com/a/36489156

**Comment by n-valchev** [comment]
Just change the default config's applicationId to `com.example` in the [build.gradle](https://github.com/react-native-maps/react-native-maps/blob/e67f773da8ce4a955fb2e4d69e6ca867c1ff95d8/example/android/app/build.gradle#L132):
```gradle
    defaultConfig {
        applicationId "com.example"
```

**Comment by Simon-TechForm** [comment]
Fixed by #4002.
